### PR TITLE
added shared isntances to the cli

### PR DESCRIPTION
**Pull request by tylerfong**

### CODE REVIEW

File: pkg/store/workspace.go [hunk 1]
```diff
@@ -169,6 +170,12 @@ func (s AuthHTTPStore) GetWorkspaces(organizationID string, options *GetWorkspac
 		for _, w := range workspaces {
 			if w.CreatedByUserID == options.UserID {
 				userWorkspaces = append(userWorkspaces, w)
+			} else {
+				if w.AdditionalUsers != nil {
```
Comment (theFong) [inline pkg/store/workspace.go]: make a function called w.CanShow(userID)

File: pkg/cmd/ls/ls.go [hunk 2]
```diff
@@ -447,9 +447,13 @@ func displayWorkspacesTable(t *terminal.Terminal, workspaces []entity.Workspace)
 	}
 	ta.AppendHeader(header)
 	for _, w := range workspaces {
+		isShared := ""
+		if w.CreatedByUserID != userID {
```
Comment (theFong) [inline pkg/cmd/ls/ls.go]: make a function called w.IsShared(userID)

**Review by theFong — CHANGES_REQUESTED** [review]

File: pkg/store/workspace.go (hunk 1 above)
Comment (tylerfong) [reply pkg/store/workspace.go]: fixed

File: pkg/cmd/ls/ls.go (hunk 2 above)
Comment (tylerfong) [reply pkg/cmd/ls/ls.go]: fixed

**Pull request merged**